Makefile

```diff
@@ -20,6 +22,25 @@ test: install src/imods.tar
 	yarn run vue-cli-service electron:serve $(SERVE_FLAGS)
 	# yarn dev
 
+webapp: install webAppModTrees.json
```

Comment:
some WIP make targets; these require some weird tooling things (like the full asset pack) and aren't expected to work yet